README.md

```diff
@@ -1,2 +1,234 @@
+
+The **Issues** and **Projects** tabs are disabled for this repository as this is part of the Ballerina library. To
+report bugs, request new features, start new discussions, view project boards, etc., visit the Ballerina
+library [parent repository](https://github.com/ballerina-platform/ballerina-library).
+
+This repository only contains the source code for the package.
+
+## Build from the source
+
+### Prerequisites
+
+1. Download and install Java SE Development Kit (JDK) version 17. You can download it from either of the following
+   sources:
+   
+    * [Oracle JDK](https://www.oracle.com/java/technologies/downloads/)
+    * [OpenJDK](https://adoptium.net/)
+
+   > **Note:** After installation, remember to set the `JAVA_HOME` environment variable to the directory where JDK was
+   installed.
+
+2. Download and install [Ballerina Swan Lake](https://ballerina.io/).
+
+3. Download and install [Docker](https://www.docker.com/get-started).
+
+   > **Note**: Ensure that the Docker daemon is running before executing any tests.
+
+4. Download `sapidoc3.jar` and `sapjco3.jar` middleware libraries from the SAP support portal and copy those libraries
+   to `native/libs` folder.
+
+5. Download the native SAP JCo library and copy it to the JAVA Class path. (This is for running test cases and examples
+   ONLY)
+
+   | OS      | Native SAP jcolibrary |
+   |---------|-----------------------|
+   | Linux   | `libsapjco3.so`       |
+   | Windows | `sapjco3.dll`         |
+   | MacOS   | `libsapjco3.dylib`    |
+
+### Build options
+
+Execute the commands below to build from the source.
+
+1. To build the package:
+
+   ```bash
+   ./gradlew clean build
+   ```
+
+2. To run the tests:
+
+   ```bash
+   ./gradlew clean test
+   ```
+
+3. To build the without the tests:
+
+   ```bash
+   ./gradlew clean build -x test
+   ```
+
+4. To run tests against different environment:
+
+   ```bash
+   ./gradlew clean test 
+   ```
+
+5. To debug package with a remote debugger:
+
+   ```bash
+   ./gradlew clean build -Pdebug=<port>
+   ```
+
+6. To debug with the Ballerina language:
+
+   ```bash
+   ./gradlew clean build -PbalJavaDebug=<port>
+   ```
+
+7. Publish the generated artifacts to the local Ballerina Central repository:
+
+    ```bash
+    ./gradlew clean build -PpublishToLocalCentral=true
+    ```
+
+8. Publish the generated artifacts to the Ballerina Central repository:
+
+   ```bash
+   ./gradlew clean build -PpublishToCentral=true
+   ```
+
+## Contribute to Ballerina
+
+As an open-source project, Ballerina welcomes contributions from the community.
+
+For more information, go to
+the [contribution guidelines](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md).
+
+## Code of conduct
+
+All the contributors are encouraged to read the [Ballerina Code of Conduct](https://ballerina.io/code-of-conduct).
+
+## Useful links
+
+* For more information go to the [`sap` package](https://lib.ballerina.io/ballerinax/sap/latest).
```

Comment:
I guess the README is same as SAP?

Reply:
Fixed with [8ac58e1](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/2/commits/8ac58e1dc2c1ee6f3313fc0f7f72cb045e2e1f6b).